src/java/org/apache/cassandra/db/virtual/AccordVirtualTables.java

```diff
@@ -42,22 +46,24 @@ public static Collection<VirtualTable> getAll(String keyspace)
         if (!DatabaseDescriptor.getAccordTransactionsEnabled())
             return Collections.emptyList();
 
-        return Arrays.asList(
-        new Epoch(keyspace)
+        return List.of(
+            new Epoch(keyspace),
+            new CommandStoreCache(keyspace)
         );
     }
 
     @VisibleForTesting
     public static final class Epoch extends AbstractVirtualTable
```

Comment:
not for this patch... but wondering if we should drop this table... I added it as a hack when we didn't have TCM...

Reply:
Yeah, I think I'll just clean it up/remove it while I'm here.